README.md

```diff
@@ -46,17 +50,15 @@ As an example, a map should be able to provide different spawn points as the mat
 
 2. Compile the latest version of TGM or download it from our [Jenkins](https://jenkins.bennydoesstuff.me/job/TGM/).
  
-3. Create a `Maps` folder in the root folder and insert a supported TGM map. Make sure you also include a rotation.txt with the maps you would like to be present in the rotation.
-    - You can download our Maps folder as a reference on the Maps repo located [here](https://github.com/WarzoneMC/Maps).
-    - If you would like to access multiple repositories or simply change the repository location, you can change the repository location found in the `plugins/TGM/config.yml` file.
- 
-4. (Optional) Install WorldEdit to enable the Teleport Tool. 
```

Comment:
Shouldn't we keep the whole install WorldEdit? It's still required to make the teleport tool functional so wouldn't it make sense to keep that and include it in the readme?

Reply:
Oh I thought that was broken and replaced with a menu. Shows how long I've been gone :P